src/components/ga4/EventBuilder/ValidateEvent/validator.ts

```diff
@@ -0,0 +1,29 @@
+import { Draft07 } from "json-schema-library"
+
+// example:
+// const jsonSchema = new Draft07(myJsonSchema);
+// const errors: JSONError[] = jsonSchema.validate(myData);
+
+type schemaObject = {
+    [key:string]: any;
+}
+
+export class Validator {
+    schema: schemaObject 
+    validator: any
+
+    constructor(
+        schema: schemaObject
+    ) {
+        this.schema = schema
+        this.validator = new Draft07(schema)
```

Comment:
It is possible to add a comment on what exactly is happening here?

Reply:
Added!